exercises/practice/tournament/.docs/instructions.md

```diff
@@ -21,9 +20,12 @@ What do those abbreviations mean?
 - L: Matches Lost
 - P: Points
 
-A win earns a team 3 points. A draw earns 1. A loss earns 0.
+A win earns a team 3 points.
+A draw earns 1.
+A loss earns 0.
 
-The outcome should be ordered by points, descending. In case of a tie, teams are ordered alphabetically.
+The outcome should be ordered by points, descending.
```

Comment:
This wording has not changed, _should_ is different than _must_, but now there is a test that is enforcing that this must happen.

Reply:
https://github.com/exercism/problem-specifications/pull/2135